SPECS/csi-driver-lvm/csi-driver-lvm.spec

```
@@ -0,0 +1,41 @@
+Summary:        Container storage interface for logical volume management
+Name:           csi-driver-lvm
+Version:        0.4.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/metal-stack/csi-driver-lvm
+#Source0:       https://github.com/metal-stack/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}-govendorcached.tar.gz
```

Nit-pick: this is the preferred way of doing this now (work better with our tooling and is easier to automatically parse). I think we should document it better, sorry about the confusion.

```suggestion
Source0:        https://github.com/metal-stack/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}-govendorcached.tar.gz
```

---

Ok, will revise it.